client/src/components/Checkout.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+class Checkout extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      rooms: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('/rooms/1/reservations') //hard coded TODO refactor to be dynamic
+      .then((response) => {
+        return response.json();
+      })
+      .then((myJSON) => {
+        this.setState({ rooms: (myJSON) });
```

Comment:
@Pfdiamond13 @fradleybrancis Variable names should describe the content of data, not the format ("myJSON")